### PR TITLE
Update comfig.cfg

### PR DESCRIPTION
**Pull request by Tiagoquix**

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -1181,17 +1181,17 @@ filesystem_max_stdio_read 64 // Use a higher max read cap (64MB)
 //mod_load_anims_async 1 // Async animation loading
 //mod_load_mesh_async 1 // Async mesh loading
 //mod_load_vcollide_async 1 // Async vcollide loading
-mod_forcetouchdata 1 // Put all model data into cache on load
-mod_forcedata 1 // Load submodel data and load lingering async loads
+//mod_forcetouchdata 1 // Put all model data into cache on load
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: This is meant to be set by default, not in modules, same with mod_forcedata.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 2]
```diff
@@ -1027,7 +1026,8 @@ alias switchconsole flip_debug
 //cc_minvisibleitems 1 // Minimum captions to shown
 //cc_predisplay_time 0 // Delay between the sound and showing the caption
 //cc_sentencecaptionnorepeat 4 // How many times a caption can repeat
-//snd_vox_captiontrace 1 // show sentence IDs when there is no caption
+//snd_vox_captiontrace 1 // Show sentence IDs when there is no caption
+//cc_lang brazilian // Force a specific closed caption language
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: `cc_lang lang`

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: config/mastercomfig/cfg/comfig/comfig.cfg (hunk 1 above)
Comment (Tiagoquix) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: So this will be the value 1 with mod_support off and on?

Comment (mastercoms) [reply config/mastercomfig/cfg/comfig/comfig.cfg]: Yes, to increase caching.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**